english/dream/tender-defender/tender-defender.md

```diff
@@ -25,4 +25,4 @@ DO BE: ready for anything
 DON'T BE: completely oblivious or totally anxious
 
 If you can do all these things, you will be on your way to becoming 
-[what women want](http://google.com)
\ No newline at end of file
+[what women want](https://www.amazon.com/dp/B00RTY0FKK/ref=dp-kindle-redirect?_encoding=UTF8&btkr=1)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.